main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+    for nickname, options in data.items():
+        guild = None
+        if options.get("guild"):
+            guild, _ = Guild.objects.get_or_create(
+                name=options.get("guild").get("name"),
+                description=options.get("guild").get("description"),
+            )
+
+        race, _ = Race.objects.get_or_create(
+            name=options.get("race").get("name"),
+            description=options.get("race").get("description"),
+        )
+
+        for skill in options.get("race").get("skills"):
```

Comment:
In case you use chaining `.get()` it would be beneficial to pass a default argument of an empty dict and list to prevent errors
```suggestion
        for skill in options.get("race", {}).get("skills", []):
```